specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/artifacts.json

```diff
@@ -102,8 +102,8 @@
           }
         ],
         "responses": {
-          "200": {
-            "description": "OK.",
+          "202": {
```

Comment:
I need more context here, does it mean it was wrong to say that the server was returning a 200? Or did you change the API to return a 202? Could you describe a little more if your 202 is pollable (have a Location header, etc.)

We can have this discussion by email if you anticipate it to be long.
Thanks!

Reply:
We got the Swagger spec for our RP devs. According to our tests, the Swagger spec is incorrect. The server is actually returning 202 rather than 200. The response is pollable, we do have a Location header in the response. Let me know if you need more info.